src/hooks/usePopoverRef/index.tsx

```diff
@@ -37,7 +37,9 @@ export default <T extends HTMLElement>(
         : useClickToggleController<T>();
```

Comment:
setIsPopoverOpen only exsist on the useClickToggleController. Caused the useEffect to crash when hover was true